sickbeard/clients/deluged_client.py

```diff
@@ -143,48 +119,48 @@ def test(self):
             self.connect()
         except Exception:
             return False
-        return True
+        else:
+            return True
 
     def add_torrent_magnet(self, torrent, options, torrent_hash):
-        torrent_id = False
         try:
             self.connect()
             torrent_id = self.client.core.add_torrent_magnet(torrent, options).get()  # pylint:disable=no-member
             if not torrent_id:
                 torrent_id = self._check_torrent(torrent_hash)
         except Exception:
             return False
+        else:
+            return torrent_id
         finally:
             if self.client:
                 self.disconnect()
 
-        return torrent_id
-
     def add_torrent_file(self, filename, torrent, options, torrent_hash):
-        torrent_id = False
         try:
             self.connect()
             torrent_id = self.client.core.add_torrent_file(filename, b64encode(torrent), options).get()  # pylint:disable=no-member
             if not torrent_id:
                 torrent_id = self._check_torrent(torrent_hash)
         except Exception:
             return False
+        else:
```

Comment:
placing the else with return here, it will never hit finaly. Is that the intention?